### ISSUE TITLE
🔥🔥🔥 Replace CountryField in survey_i18n

### ISSUE THREAD

**Issue by dheavy** [medialab/trustlab#2]
It causes a bug in the admin panel that turns into a nasty memory leak crashing the server.
Replace it. i18n should follow if possible.

**Comment by dheavy** [comment]
### Expected behavior
[`django-countries`](https://github.com/SmileyChris/django-countries) renders a country list in `select` element when you use it as a field in a model. Its data should be visible within the **results** tab of the **admin panel**. 

### Actual behavior
But any app implementing `django-countries` (i.e. `survey_i18n`) crashes the entire platform when user views its results on the dedicated tab.
```python
"TypeError: Country([...snippet removed...]) is not JSON serializable"
```
**Results** tab will show a _"failed to connect to server"_ error. It also triggers a memory leak.

### Steps to reproduce the behavior
Display the **results** tab of an experiment that embeds at least one `CountryField` in its models.

**Comment by dheavy** [comment]
Fixed ([68f972f](https://github.com/medialab/trustlab/commit/68f972fcaff5bcaa6064e1f917d28a68387cea3b)).
